### PR TITLE
Application: don't simulate until we are ready to render

### PR DESCRIPTION
**Pull request by hyperlogic**
This should prevent the issue where we would sometimes simulate more often than we were rendering.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1321,10 +1321,6 @@ void Application::paintGL() {
         _lastFramesPerSecondUpdate = now;
     }
 
-    if (_isGLInitialized) {
-        idle(now);
-    }
-
```
Comment (hyperlogic) [inline interface/src/Application.cpp]: Remove idle from paintGL

File: interface/src/Application.h [hunk 2]
```diff
@@ -509,7 +509,6 @@ private slots:
     int _avatarAttachmentRequest = 0;
 
     bool _settingsLoaded { false };
-    bool _pendingPaint { false };
```
Comment (hyperlogic) [inline interface/src/Application.h]: We no longer need the _pendingPaint flag.

**Comment by hyperlogic** [comment]
https://app.asana.com/0/26225263936266/92650435482459

**Comment by hifi-gustavo** [comment]
Build console output for commit: 08c16beff26e4eb93b10a7d459309606fa4c389a available at: https://gist.github.com/5de3cc9b9c44f1775ed2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7206.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7206.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e5c81f66a54c8c9603fa03bd5e822d8aa67de055 available at: https://gist.github.com/5de3cc9b9c44f1775ed2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7206.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7206.dmg

**Comment by bluetoe** [comment]
Sim rate reported in Stats seems now to closely match render rate. Unless I create a ton of physical stuff, which is to be expected.

**Comment by bluetoe** [comment]
One thing I wonder about is when HiFi is not main focus, i.e. when minimized or other application or window is on top, the render rate drops, which is what is intended, but so does the Stat reported Simrate.

Could this make problems for physical stuff?.. If you click to look at Skype or something will this affect your version of physics in HiFi?

**Comment by PhilipRosedale** [comment]
Tested and confirmed working in multiple locations.  Sim rate goes as high as render/present, no higher.  Release goes up to 120.

**Pull request merged**